Studentmediene/kapina-backend
# feat: add Spotify plugin to CKEditor

BLOCKED BY: https://github.com/Studentmediene/kapina-backend/pull/63 (Merge that one first)

Adds a Spotify plugin (button with dialog) to the CKEditor when editing articles. 

How to QA:
- Create article, check that Spotify button exists in the editor. 
- Use the button to create a playlist
